src/Symfony/Component/Notifier/Exception/FlashMessageMappingException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Exception;
+
+/**
+ * @author Ben Roberts <ben@headsnet.com>
+ */
+class FlashMessageMappingException extends LogicException
+{
+    public function __construct(string $importance, string $mappingClass)
+    {
+        $message = sprintf(
+            'The Notifier flash message mapper "%s" does not support an importance value of "%s"',
```

Comment:
```suggestion
            'The "%s" Notifier flash message mapper does not support an importance value of "%s".',
```

can you also move the whole code on one-line?